src/test/scala/io/iohk/ethereum/network/p2p/messages/NewBlockSpec.scala

```diff
@@ -12,10 +12,17 @@ import org.scalatest.funsuite.AnyFunSuite
 
 class NewBlockSpec extends AnyFunSuite with ScalaCheckPropertyChecks with ObjectGenerators with SecureRandomBuilder {
 
-  val chainId = Hex.decode("3d").head
+  val chainId_1 = 1
+  val chainId_212 = 212
```

Comment:
nitpick: mixing camelCase with snake_case. Shall we name them chainId1 and chainId212?

Reply:
fixed